### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ppmtscory**
Changed example urls to example.com domain.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,10 +93,10 @@ These access tokens are in turn used by your application to call certain PayPal
 the **scopes** you request during token creation. The first time you want to generate an access token for an account,
 you must send them to PayPal.com to login and consent to the scopes you request. This is accomplished by the */toPayPal*
 endpoint in [server.js](server.js) and in turn the *redirect* method in this module (which you would use if integrating into your own node.js server). For example, for the
-client id ABCDEFG which is registered with developer.paypal.com as having a return url of http://abcdefg.com/returnFromPayPal, you would send your merchant to
+client id ABCDEFG which is registered with developer.paypal.com as having a return url of http://example.com/returnFromPayPal, you would send your merchant to
```
Comment (snehanshuashar) [inline README.md]: Can we change ABCDEFG client id also to something more relevant?
EXAMPLEID for the client id
exampledomain for the url

Comment (ppmtscory) [reply README.md]: Any suggestions?  Having this as a sample client ID shouldn't matter.  The only other thing I'd think of would be all X's or something like that.

Comment (ppmtscory) [reply README.md]: BT specifically requested example.com for the domains as it's not an available domain for somebody to purchase and it's uses solely for these purposes.

Comment (ppmtscory) [reply README.md]: I'm cool with exampleid for the client ID

Comment (ppmtscory) [reply README.md]: made the changes

**Pull request merged**